src/popup/popup.html

```diff
@@ -17,29 +27,30 @@
       }
 
       ul li {
-        padding: 10px;
-        margin: 2px;
+        padding: 5px;
+        margin: 0px;
+      }
+
+      ul li:hover {
+        background-color: #F5F5F5;
       }
 
       a {
         text-decoration: none;
+        color: #4682B4;
+        display:block
       }
 
-      a:hover {
-        text-decoration: underline;
-      }
     </style>
   </head>
-  <body style="width: 300px;">
-    <h3>Current language: <span data-current-lang>loading...</span></h3>
-
-    <hr>
+  <body style="width: 280px;">
+    <h4>Current language: <span data-current-lang>loading...</span></h4>
 
     <ul>
-      <li><a href="#" data-lang>English (Original)</a></li>
-      <li><a href="#" data-lang="fr">Français</a></li>
-      <li><a href="#" data-lang="de-DE">Deutsch (Deutschland)</a></li>
-      <li><a href="#" data-lang="ja">日本語</a></li>
+      <li><a href="#" data-lang data-display-lang="English">English (Default)</a></li>
+      <li><a href="#" data-lang="fr" data-display-lang="Français">Français</a></li>
+      <li><a href="#" data-lang="de-DE" data-display-lang="Deutsch">Deutsch (Deutschland)</a></li>
+      <li><a href="#" data-lang="ja" data-display-lang="日本語">日本語</a></li>
```

Comment:
I think we can avoid complexity by getting rid of `data-display-lang` in favor of the inner value. That way the displayed language will always be what the user actually clicked.